components/automate-gateway/api/compliance/reporting/stats/stats.proto

```diff
@@ -236,7 +284,7 @@ message Profile {
 message ProfileList {
 	// Name of the profile.
 	string name = 1;
-	// SHA id of the profile.
+	// SHA ID of the profile.
 	string id = 2;
 	// Total number of failed nodes that executed the profile.
```

Comment:
Is this the same as "the total number of times this profile failed"? Or does it include nodes that ran the profile successfully, but might have failed on other profiles?" (I forget, is node:profile 1:1?)

Reply:
1 node: many profiles

i...think it's "the total number of times this profile failed"...but i'm not 100% on that. i'll double check and follow up

1.

Reply:
ok ya it's the total number of times this profile failed

Reply:
`total number of nodes that failed this profile` -- more active than the profile seeming to fail on its own.

Reply:
i'm gonna merge this one first so i'll get that change in the nodes proto one